src/ctfp-print-agda.tex

```diff
@@ -0,0 +1,3 @@
+\input{opt-agda}
+\input{ctfp-print}
+\input{postamble}
```

Comment:
I believe this and the remaining files of the PR are also required for compiling the agda version of ctfp (with the command `nix build .#ctfp-agda`)